src/unitxt/llm_as_judge.py

```diff
@@ -65,12 +65,12 @@ def verify(self):
             )
 
         if isinstance(self.inference_model, OpenAiInferenceEngine):
-            if self.format and type(self.format) is not SystemFormat:
-                raise ValueError(
-                    "Error in 'LLMAsJudge' metric. Inference model 'OpenAiInferenceEngine' does "
-                    "not support formatting. Please remove the format definition from the recipe"
-                    " (OpenAi Chat API take care of the formatting automatically)."
-                )
+            # if self.format and type(self.format) is not SystemFormat:
```

Comment:
change it to be either None, formats.empty or formats.chat_api

Reply:
@OfirArviv - the format is the artifact, not the string. How can I check the string from the artifacrt?

Reply:
@OfirArviv - to be more clear, I can check if the format is a chat api format instance, but don't know how to check if a system format is generated from "formats.empty" - this is probably where this bug emerged from in the first place